src/app/app-interactive-map/components/app-build/app-build.component.ts

```diff
@@ -248,9 +249,14 @@ export class AppBuildComponent implements OnInit, AfterViewInit {
           this.condition = event;
           this.store.dispatch(new SetOperations(operations, this.method, this.experiment, this.condition,
             strain.organism_id, selectedModel.id, condition));
-        }, (error) => {
+        }, (error: HttpErrorResponse) => {
           if (error) {
-            this.store.dispatch(new LoadingError());
+            const dialogConfig = new MatDialogConfig();
+            dialogConfig.data = error.error.errors.join('<br>');
```

Comment:
Does this encode html characters right? The response text is not guaranteed to be html safe.